plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHivePlugin.java

```diff
@@ -228,6 +228,84 @@ public void testSystemAccessControl()
         connector.shutdown();
     }
 
+    @Test
+    public void testHttpMetastoreConfigs()
+    {
+        ConnectorFactory connectorFactory = getHiveConnectorFactory();
+
+        Connector connector = connectorFactory.create(
+                "test",
+                ImmutableMap.<String, String>builder()
+                        .put("hive.metastore.uri", "https://localhost:443")
+                        .put("hive.metastore.http.client.bearer-token", "token")
+                        .put("hive.metastore.http.client.additional-headers", "key:value")
+                        .put("hive.metastore.http.client.authentication.type", "BEARER")
+                        .buildOrThrow(),
+                new TestingConnectorContext());
+        connector.shutdown();
```

Comment:
No need to define `connector` variable. We can chain `shutdown()` method. Same for others.